charm/ksql/metadata.yaml

```diff
@@ -0,0 +1,21 @@
+name: ksql
+summary: Streaming SQL engine that enables real-time data processing against Apache Kafka.
+maintainer: Khanh Nguyen <khanh.nguyen@canonical.com>
+description: |
+  Provides an easy-to-use, yet powerful interactive SQL interface for stream processing on Kafka, without the need to write code in a programming language such as Java or Python. 
+  This is intended to be use for internal projects for Canonical Ltd. This charm is not intended for community use.
```

Comment:
We can remove this line I think. I imagine some of our partners might want to use this charm, and might even contribute to it later on.